README.md

```diff
@@ -25,6 +25,142 @@ const schema = formSchema.getFormSchema();
 formSchema.removeField('some-great-field');
 ```
 
+## Form Validation
+
+You can write multiple validation adapters for different purposes. The idea behind this design
+decision is you can create a validation adapter that only validates the form schema but doesn't
+care about values. That way you use the form schema validation adapter to validate that the
+user created the schema correctly (text fields require a label) and the form value validator
```

Comment:
could use an e.g. at the start of that parenthetical, unless I'm reading it wrong.